code/modules/reagents/reagent_containers/food/drinks/drinkingglass.dm

```diff
@@ -211,7 +211,7 @@
 					icon_state = "beepskysmashglass"
 					item_state = "beepskysmashglass"
 					name = "\improper Beepsky Smash"
-					desc = "Heavy, hot and strong. Just like the iron fist of the LAW."
+					desc = "Heavy, hot and strong. Best drank with your hands behind your back."
```

Comment:
"Best drank"?

Reply:
`This is best drank with your hands behind your back.`

Just with `This is` omitted.

English is strange.

Reply:
It might sound less awkward if the imperative is used.

`Drink it with your hands behind your back.`

Reply:
'best enjoyed'

Reply:
I'll be frank, so far none of the suggestions have managed to sway me from the original wording here. It's in reference to chugging drinks without hands as well as the police telling people to put their hands behind their back.

Reply:
I don't think anyone was confused about the nature of the reference.

Reply:
Best enjoyed with your hands behind your back

Reply:
It is done.